website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -26,6 +26,14 @@ to `true`. In prior Consul versions (1.10.x through 1.11.x), the config defaulte
 well as the flag will be removed in upcoming Consul 1.13. We recommend changing your instrumentation to use 1.10 and later
 style `consul.api.http...` metrics and removing the configuration flag from your setup.
 
+### Nomad Namespace Incompatibility
+
+Consul 1.12.0 Enterprise introduced a bug that prevents Nomad Enterprise from
+removing services from non-default Consul namespaces.
+
+Nomad Enterprise users should *not* upgrade to Consul Enterprise 1.12.0 and
```

Comment:
I think we can soften this a little more and talk about a "change" (instead of a "bug") that will result in this behavior. We can say that we recognize that the change may negatively impact Nomad uses, so will address it in a future release. I think it's good to be transparent and proactively let users know that something may not work as intended, but we don't want to create the impression that we're not careful. Interested to hear other's opinions.

Reply:
But it is a bug.